src/Vgrep/Environment/Config/Sources/File.hs

```diff
@@ -333,7 +333,7 @@ mapMKeys f = fmap M.fromList . M.foldrWithKey go (pure [])
         xs <- mxs
         pure ((k', x) : xs)
 
-parseChord :: Monad m => String -> m Key.Chord
+parseChord :: MonadFail m => String -> m Key.Chord
```

Comment:
This needs a bit of CPP for backward compatibility.

Reply:
4.9.0 dates back to 2016 (GHC 8.0.1), so I'm a bit inclined to drop support for GHC 7.x. after almost five years… What do you think?

Reply:
And BTW, I think this is missing an `import Control.Monad.Fail (MonadFail)`

Reply:
Removed the CPP and fixed the build.